scanners/a11y.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from scanners import utils
+import json
+import os
+
+import shlex
+from subprocess import Popen, PIPE
+import json
+
+def get_exitcode_stdout_stderr(cmd):
+    """
+    Execute the external command and get its exitcode, stdout and stderr.
+    """
+    args = shlex.split(cmd)
+
+    proc = Popen(args, stdout=PIPE, stderr=PIPE)
+    out, err = proc.communicate()
+    exitcode = proc.returncode
+
+    return exitcode, out, err
```

Comment:
Can we set `stdout` and `stderr` to print out to the usual place?

Do the [arguments in this line](https://github.com/unitedstates/inspectors-general/blob/master/inspectors/utils/utils.py#L299) point in a good direction:

```python
subprocess.Popen(["pdftotext", "-v"], shell=False, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT).communicate()
```